searchcore/src/vespa/searchcore/proton/server/proton.cpp

```diff
@@ -432,10 +434,9 @@ Proton::~Proton()
     if (_fs4Server) {
         _fs4Server->shutDown();
     }
-    while (!_documentDBMap.empty()) {
-        const DocTypeName docTypeName(_documentDBMap.begin()->first);
-        removeDocumentDB(docTypeName);
-    }
+    size_t numCores = std::max(1u, std::thread::hardware_concurrency());
```

Comment:
I think we should use HwInfo.cpu().cores() instead. That is based on proton.cfg, and if nothing is set we actually use std::thread::hardware_concurrency(). An instance of HwInfo is found in BootstrapConfig.

Reply:
I agree. Fixed